docs/apis/http/tus_upload.md

```diff
@@ -13,6 +13,35 @@ In situations where file uploads might be interrupted due to network issues, bro
 tus ensures that uploads can be resumed from the point of failure without losing data.
 This documentation shows some basic examples, refer [tus official site](https://tus.io/protocols/resumable-upload) for more details.
 
+## Supported tus Features
+
+The backend announces certain tus features to clients. WebDAV responses come with tus HTTP headers for the offical tus features, and additional, ownCloud specific features are announced via the capabilities endpoint (e.g. `https://localhost:9200/ocs/v1.php/cloud/capabilities?format=json`).
```

Comment:
The WebDAV response headers are mentioned here. Which WebDAV calls return them, and which headers are this and what do they mean?

If you do not want to answer that here, I would not mention these headers at all.

Reply:
We link to the tus docs in the paragraph above. Explanation there…